src/monsters.cpp

```diff
@@ -879,6 +879,11 @@ MonsterType* Monsters::loadMonster(const std::string& file, const std::string& m
 		} else {
 			std::cout << "[Error - Monsters::loadMonster] Missing health max. " << file << std::endl;
 		}
+
+		if (mType->info.health > mType->info.healthMax) {
+			mType->info.health = mType->info.healthMax;
+			std::cout << "[Warning - Monsters::loadMonster] Health now greater than health max." << file << std::endl;
```

Comment:
Health now "is"/"was" greater than health max?

Reply:
I wrote it like this because all the other warnings are also written the same way.
Just tell me if you still want me to make the change you suggest and I will.